### PR TITLE
Removing Kubernetes version from Antrea install

### PR DESCRIPTION
**Pull request by knabben**
Fixes #https://github.com/kubernetes-sigs/sig-windows-dev-tools/issues/146
Fixes #https://github.com/kubernetes-sigs/sig-windows-dev-tools/issues/138

### CODE REVIEW

File: sync/linux/controlplane.sh [hunk 1]
```diff
@@ -141,6 +141,80 @@ EOF
 kubeadm token create --print-join-command >> /var/sync/shared/kubejoin.ps1
 
 sed -i 's#--token#--cri-socket "npipe:////./pipe/containerd-containerd" --token#g' /var/sync/shared/kubejoin.ps1
+
+cat << EOF > kube-proxy-and-antrea.yaml
```
Comment (dougsland) [inline sync/linux/controlplane.sh:145]: Not sure if this is a space thing, looks like this code already exists?

**Review by dougsland — COMMENTED** [review]
lgtm

File: sync/linux/controlplane.sh [hunk 2]
```diff
@@ -40,7 +32,7 @@ fi
 
 kubernetes_version=${1}
 k8s_kubelet_node_ip=${2}
-k8s_linux_apiserver="ci/latest-${kubernetes_version}"
+k8s_linux_apiserver="stable-${kubernetes_version}"
```
Comment (knabben) [inline sync/linux/controlplane.sh:35]: https://github.com/kubernetes/kubernetes/blob/ea0764452222146c47ec826977f49d7001b0ea8c/cmd/kubeadm/app/util/version.go#L59

Comment (dougsland) [reply sync/linux/controlplane.sh:35]: good one, what's the version you are getting now ?

File: sync/linux/controlplane.sh (hunk 1 above)
Comment (knabben) [reply sync/linux/controlplane.sh:137]: the RBAC was removed in the other PR, but still required for Antrea, letting here for now.

File: sync/linux/controlplane.sh (hunk 2 above)
Comment (knabben) [reply sync/linux/controlplane.sh:35]: the stable version of the minor.

File: sync/linux/controlplane.sh [hunk 3]
```diff
@@ -115,7 +107,7 @@ networking:
   podSubnet: "100.244.0.0/16"
 EOF
 
-sudo kubeadm init --config=/var/sync/shared/kubeadm.yaml --v=6
+sudo kubeadm init --config=/var/sync/shared/kubeadm.yaml --v=6 --ignore-preflight-errors=All
```
Comment (dougsland) [inline sync/linux/controlplane.sh:110]: Maybe add a comment why adding  --ignore-preflight-errors=All ? Is it because dockershim ? Swap Error?

**Review by dougsland — COMMENTED** [review]
lgtm

File: sync/linux/smoke-test.yaml [hunk 4]
```diff
@@ -56,6 +56,10 @@ spec:
       labels:
         app: whoami-windows
     spec:
+      securityContext:
```
Comment (knabben) [inline sync/linux/smoke-test.yaml:59]: https://github.com/antrea-io/antrea/issues/2774

workaround for `containerd github.com/containerd/containerd v1.5.5 72cec4be58a9eb6b2910f5d10f1c01ca47d231c0`

Comment (knabben) [reply sync/linux/smoke-test.yaml:59]: this is failing for Calico as well. mostly some issue with containerd at this point. trying a newer version. keeping hostProcess and hostNetwork if it did not get fixed until the final solution

Comment (dougsland) [reply sync/linux/smoke-test.yaml:59]: Please correct me if I am wrong:
- HostProcess is Calico stuff, should be ignored by Antrea ?
- When using HostProcess require to add runAsUserName like runAsUserName: "NT AUTHORITY\\FooBar" ?

I would suggest to put this YAML in a different file and use envsubs (if calico is true add these bits)

**Pull request merged**